apps/wallet/src/ui/app/pages/home/nft-transfer/index.tsx

```diff
@@ -79,25 +93,30 @@ function NftTransferPage() {
     const handleOnClearSubmitError = useCallback(() => {
         setSendError(null);
     }, []);
-    const { loading } = useObjectsState();
-    return (
-        <div className={cl(st.container, { 'items-center': loading })}>
-            <Loading loading={loading}>
-                {selectedNft && objectId && hasPublicTransfer(selectedNft) ? (
-                    <>
-                        <PageTitle
-                            title="Send NFT"
-                            back={`/nft-details?${new URLSearchParams({
-                                objectId,
-                            }).toString()}`}
-                        />
-                        <div className={st.content}>
-                            <NFTDisplayCard
-                                nftobj={selectedNft}
-                                wideView
-                                size="sm"
-                            />
 
+    const closeSendToken = useCallback(() => {
+        navigate('/');
+    }, [navigate]);
+
+    return (
+        <Overlay
+            showModal={showModal}
```

Comment:
This overlay is static in this page right? we show it always and if we want to close it we navigate away. If so why do we need this variable, we could just pass true here